machines/README.md

```diff
@@ -15,12 +15,14 @@ Machine.machine.include parameters
                                             If this flag is set to zero, the manycore can only operate in NO-DRAM mode,
                                             meaning that the vcache is only used as block memory.
 - BSG_MACHINE_MAX_EPA_WIDTH             =   Width of word address on the mesh network.
-- BSG_MACHINE_MEM_CFG                   =   e_vcache_non_blocking_axi4_nonsynth_mem
+- BSG_MACHINE_MEM_CFG                   =   e_vcache_non_blocking_axi4_nonsynth_mem (look at bsg_manycore_mem_cfg_pkg.v)
```

Comment:
Why not: "Memory Configuration for manycore machine (e.g. infinite memory, see bsg_manycore_mem_cfg_pkg.v for valid values)"